packages/test-runner/src/config/parseConfig.ts

```diff
@@ -188,8 +188,10 @@ export async function parseConfig(
       // we can improve this by relying only on groups inside the test runner itself
       if (groupConfig.files == null) {
         groupConfig.files = finalConfig.files;
+        groupConfig.allowPassWithoutTests = finalConfig.allowPassWithoutTests;
       }
       finalConfig.files = undefined;
+      finalConfig.allowPassWithoutTests = undefined;
```

Comment:
what is the reason to set it as `undefined`? isn't it more clear to have it always as a boolean and default to `false`?